packages/prisma/zod-utils.ts

```diff
@@ -81,6 +81,12 @@ export const EventTypeMetaDataSchema = z
     disableSuccessPage: z.boolean().optional(),
     disableStandardEmails: z
       .object({
+        all: z
+          .object({
+            host: z.boolean().optional(),
+            attendee: z.boolean().optional(),
+          })
+          .optional(),
```

Comment:
We now have a new option to disable all emails while still keeping the old option for just confirmation emails.